qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1332,17 +1334,25 @@ def jobs(
         return [self._decode_job(job) for job in job_responses]
 
     def delete_job(self, job_id: str) -> None:
-        """Delete a runtime job.
+        """(DEPRECATED) Delete a runtime job.
```

Comment:
Why was this deprecated? Deleting a job works fine in the cloud stack, there is a documented API for it and internal integration tests that run against it daily.

https://quantum.cloud.ibm.com/docs/en/api/qiskit-runtime-rest/tags/jobs

<img width="1920" height="878" alt="image" src="https://github.com/user-attachments/assets/8a8f2943-3234-4d4d-9fdd-51f19c14bfff" />